incubator/custom-iptables/templates/daemonset.yaml

```diff
@@ -25,6 +25,9 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           command: [ "sh", "-c", "/run.sh" ]
+          env:
+          - name: SLEEP_INTERVAL
+            value: "{{ .Values.sleepInterval }}"
```

Comment:
In my experience with helm and integer translation, `{{ .Values.sleepInterval | quote }}` works more consistently, but I haven't tried this method in a long time, so it might be fine.